plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageInitialization.java

```diff
@@ -60,6 +58,8 @@ class ConnectionPageInitialization extends ConnectionWizardPage implements IData
 
     private static final Log log = Log.getLog(ConnectionPageInitialization.class);
 
+    private static final String PAGE_DOCS_LINK = "https://dbeaver.com/docs/dbeaver/Configure-Connection-Initialization-Settings/";
```

Comment:
But our wiki doesn't have this page. And keep-alive info...

Reply:
It will be published soon

Reply:
https://github.com/dbeaver/tech-docs/issues/200

Reply:
Yup, this link points to nowhere. This is a no-no.